CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.IO;
+using System.Diagnostics;
+using Newtonsoft.Json;
+
+namespace CalculatorLibrary
+{
+    public class CalculatorProgram
+    {
+        JsonWriter writer;
+        private int _calUsageCount = 0;
+        //private string _histNum;
```

Comment:
Old code? In general it's best to delete old code instead of comment it out. If you need this old code for some reason then you can get that history from git instead.